ansible/playbooks/SQLclusterPlaybooks/step2.yml

```diff
@@ -0,0 +1,15 @@
+- hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_standard_eula: true
+    mssql_firewall_configure: true
+    mssql_password: "RedH@123!"
+    mssql_edition: Developer
+    mssql_debug: true
+  tasks:
+    - name: Configure SQL Server 
+      include_role:
+        name: microsoft.sql.server
+      when: mssql_ha_replica_type == 'synchronous' or mssql_ha_replica_type == 'witness'
```

Comment:
Given that step1 and step2.yml almost the same except for this line and a couple of vars, is it worth combining them both into one file with two tasks?

The tasks will still execute sequentially, so your witness/synchronous nodes will come up after the primary is configured.

Not a biggie, and def not a merge blocker, more just a style question.

Reply:
The intent was to separate these because step1 is separate task.  You deploy the database and make sure everything is working before you cluster it.